portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,78 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+/**
+*show the next picture on the screen
+ */
+ //ask for static variable because we heard that globalis not good
+const counterObj = {
+    picturesIndex: 0
+}
```

Comment:
Using an object over a primitive type here is still considered a global.
```suggestion
let picturesIndex = 0;
```